cylc/flow/tests/test_config.py

```diff
@@ -212,6 +216,35 @@ def test_family_inheritance_and_quotes(self):
                 assert 'goodbye_0_major1_minor10' in \
                        config.runtime['descendants']['SOMEFAM']
 
+    def test_missing_initial_cycle_point(self):
```

Comment:
These pytests can be run standalone outside of `TestSuiteConfig` (i.e. dedent them one level)